specification/mysql/resource-manager/readme.python.md

```diff
@@ -0,0 +1,142 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml !$(track2)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  package-name: azure-mgmt-rdbms
+  clear-output-folder: true
+  no-namespace-folders: true
+```
+
+These settings apply only when `--track2` is specified on the command line.
+
+``` yaml $(track2)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+package-name: azure-mgmt-rdbms
+no-namespace-folders: true
+```
+
+### Python multi-api
+
+Generate all API versions currently shipped for this package
+
+```yaml !$(track2)
+batch:
+  - tag: package-2020-01-01
+  - tag: package-2018-06-01
+  - tag: package-2017-12-01
+  - tag: package-2020-07-01-privatepreview
+  - tag: package-2020-01-01-privatepreview
+  - tag: package-2018-06-01-privatepreview
+  - tag: package-2017-12-01-preview
```

Comment:
Hi @mjain2 , want to make sure do we need so many tags?
I think there should be two: one for old server and one for flexibleServer.